### DISCUSSION TITLE
Is it working on Gradle 7+ ?

### DISCUSSION THREAD

**Discussion by portlek** [jreleaser/jreleaser#340] [Q&A] [answered]
I'm getting this error, idk this is an issue or not so i open into q&a if it's an issue then let me know so i can open an issue for it.
![image](https://user-images.githubusercontent.com/12835420/128599662-d7e45279-3d22-4c7f-a818-664df1f75b74.png)
https://github.com/jreleaser/jreleaser/blob/main/plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/dsl/SdkmanAnnouncerImpl.groovy#L71

it's about this change in the file. the command field haven't set.
https://github.com/jreleaser/jreleaser/commit/ae0bd2103068a935fe37aadacc9560e5e95498f6#diff-2d5de8e03016e662ea1629e0b0266efd78219daf787f682e96b46d4de7ec4843

**Comment by aalmiray** [comment, marked as answer]
Thank you for reporting the issue https://github.com/jreleaser/jreleaser/issues/341 and sending a PR https://github.com/jreleaser/jreleaser/pull/342
